docs/modules/fortiq_42XX.rst

```diff
@@ -1,23 +1,23 @@
 ********************************
-Fortiq M42BLS | Servo Module
+Fortiq 42XX | Servo Module
 ********************************
 
 .. sidebar:: Fortiq
 
     .. image:: ../_static/Fortiq_family.jpg
-        :alt: Fortiq M42BLS Family
+        :alt: Fortiq 42XX Family
 
 
-The IQ Fortiq M42BLS module is designed for industrial and robotic applications that 
+The Fortiq 42XX module is designed for industrial and robotic applications that 
 require maximum torque, precision, and efficiency with minimal vibration. IQ's unique 
 hardware design and advanced calibration techniques has allowed us to create the most 
```

Comment:
I know you didn't write this, but it should be "hardware design and advanced calibration techniques have allowed us..."